### DISCUSSION TITLE
Versione Home Assistant

### DISCUSSION THREAD

**Discussion by mippo9** [indomus/forum#2383] [HUB: Home Assistant]
buongiono,
scusate, ma da dove si vede qual'è la versione di Home Assistant installata ?
sto cercando di aggiornare all'ultima versione, la 2023.3, seguendo la guida apposita, ma ho l'impressione di non riuscire, ad esempio non vedo il pulsantino "Restart" in alto a destra, quindi penso che continui ad installare la versione precedente.
La guida propone di installare la versione "Stable", ho provato anche "Latest".

qualche suggerimento ...?

**Comment by snakuzzo** [comment]
Il tag latest non devi usarlo, perché include anche le beta.

La versione si vede sotto Impostazioni > Informazioni

Il tastino per restartare lo trovi cliccando su Impostazioni e poi menù in alto a destra (i tre puntini per intenderci). Se non lo vedi svuota la cache lato browser
